storage-proofs/Cargo.toml

```diff
@@ -48,7 +48,7 @@ log = "0.4.7"
 pretty_env_logger = "0.3.0"
 
 [features]
-default = []
+default = ["disk-trees"]
 simd = []
 asm = ["sha2/sha2-asm"]
 disk-trees = []
```

Comment:
`fil-proofs-tooling` and `filecoin-proofs` should get this feature too, which just proxies this, so there is control at that level

Reply:
actually fine to not do this for now, if we switch this on by default